core-bundle/contao/templates/backend/be_main.html.twig

```diff
@@ -0,0 +1,133 @@
+    <script src="{{ asset('js/simplemodal.min.js', 'contao-components/simplemodal') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/datepicker.min.js', 'contao-components/datepicker') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/tinymce.min.js', 'contao-components/tinymce4') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/ace.js', 'contao-components/ace') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/ext-language_tools.js', 'contao-components/ace') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/ext-whitespace.js', 'contao-components/ace') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('js/dropzone.min.js', 'contao-components/dropzone') }}" data-turbo-track="reload"></script>
+    <script src="{{ asset('backend.js', 'contao_core') }}" data-turbo-track="reload"></script>
+    <script>{{ getDateString.invoke()|raw }}</script>
+    {{ javascripts|default|raw }}
+  {% endblock %}
+</head>
+<body{{ bodyAttributes }}>
+  <a class="invisible show-on-focus" href="#main" data-turbo="false">{{ 'MSC.skipToContent'|trans }}</a>
+
+  {% block header %}
+    {% if not isPopup %}
+      <header id="header">
+        <div class="inner">
+          <h1>
+            <a href="{{ path('contao_backend') }}" accesskey="h" id="home" title="{{ home }}" data-contao--tooltips-target="tooltip">
+              <span class="app-title">Contao</span>
+              {% if badgeTitle|default %}
+                <span class="badge-title">{{ badgeTitle }}</span>
+              {% endif %}
+            </a>
+          </h1>
+          {{ headerMenu|raw }}
+        </div>
+      </header>
+
+      {% block header_outlets %}
+        {% include "@Contao/backend/component/message/_outlet.html.twig" %}
+        {% include "@Contao/backend/component/dialog/_outlet.html.twig" %}
+      {% endblock %}
+    {% endif %}
+  {% endblock %}
+
+  {% block container %}
+    <div id="container">
+
+      {% block left %}
+        {% if not isPopup %}
+          <aside id="left">
+            {{ menu|raw }}
+            <div class="version">
+              {% block version %}
+                {{ version }}<br>{{ learnMore|raw }}
+              {% endblock %}
+            </div>
+          </aside>
+        {% endif %}
+      {% endblock %}
+
+      {% block main %}
+        <main id="main" aria-labelledby="main_headline">
+          <h1 id="main_headline">{{ headline|raw }}</h1>
+          <div class="content">
+            {% if pickerMenu|default %}
+              <div id="picker-menu">
+                {{ pickerMenu|raw }}
+              </div>
+            {% endif %}
+            {% if error|default %}
+              <p class="tl_gerror">{{ error }}</p>
+            {% endif %}
+            {% block main_content %}
+              {{ main|raw }}
```

Comment:
This currently fails in the templates studio, do we need to fix that there or add a `|default` here?